src/views/GovernanceReviewTeam/IntakeReview/index.test.tsx

```diff
@@ -66,32 +70,30 @@ describe('The GRT intake review view', () => {
         year: '2020'
       }
     },
+    decisionNextSteps: '',
     businessNeed: 'The quick brown fox jumps over the lazy dog.',
     businessSolution: 'The quick brown fox jumps over the lazy dog.',
     currentStage: 'The quick brown fox jumps over the lazy dog.',
     needsEaSupport: false,
     grtReviewEmailBody: 'The quick brown fox jumps over the lazy dog.',
     decidedAt: new Date().toISOString(),
-    businessCaseId: null,
-    submittedAt: DateTime.fromISO(new Date(2020, 8, 30).toISOString())
-  };
+    submittedAt: DateTime.fromISO(new Date(2020, 8, 30).toISOString()).toISO()
+  } as SystemIntake;
```

Comment:
I know I complained about typecasting above, but I think it's ok here; to fix it, you'd have to add `__typename` properties everywhere, and that's really annoying. I've tried to find a way to exclude the `__typename` stuff from the types apollo-client's codegen spits out, haven't had any luck.